subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformBuildOperationIntegrationTest.groovy

```diff
@@ -121,15 +121,6 @@ class ArtifactTransformBuildOperationIntegrationTest extends AbstractIntegration
         setupExternalDependency()
 
         buildFile << """
-            allprojects {
-                dependencies {
-                    registerTransform(MakeGreen) {
```

Comment:
❓ Why don't we need this transform any more?

Reply:
Because the same transform is already registered via the helper call to `setupBuildWithColorTransformImplementation()` above